### PR TITLE
add 'root menu' option to end of form navigation

### PR DESCRIPTION
**Pull request by snopoke**
https://dimagi.uservoice.com/forums/194738-app-builder/suggestions/7085144-add-module-select-screen-to-end-of-form-navigation

### CODE REVIEW

**Comment by dimagimon** [comment]
:+1: No lint errors found.

**Comment by snopoke** [comment]
@amsagoff @christineawhite this was a quick and easy one. This adds an option to the end of form navigation which takes the user back to the 'root menu' (the first menu you see when you click the 'start' button). I just wanted to run the dropdown text past you. The options I thought of were:

* Root menu
* Module menu

I went with the first option.

File: corehq/apps/app_manager/tests/test_suite.py [hunk 1]
```diff
@@ -241,14 +241,24 @@ def test_form_workflow_module(self):
...
+
         app = Application.wrap(self.get_json('suite-workflow'))
         for m in [1, 2]:
             module = app.get_module(m)
             module.put_in_root = True
 
+        self.assertXmlEqual(self.get_xml('suite-workflow-module-in-root'), app.create_suite())
+
+    def test_form_workflow_root(self):
+        # app = Application.wrap(self.get_json('suite-workflow-root'))
```
Comment (benrudolph) [inline corehq/apps/app_manager/tests/test_suite.py]: nit: could get rid of the commented line

**Comment by snopoke** [comment]
@benrudolph :white_check_mark:

**Pull request merged**